src/components/Calendar/calendar.tsx

```diff
@@ -0,0 +1,98 @@
+import React, { useState } from 'react';
+import { StyleSheet, Pressable, View, Text } from 'react-native';
+import DateTimePicker, { Event } from '@react-native-community/datetimepicker';
+import { QuestionComponentProps } from 'types/types';
+
+const styles = StyleSheet.create({
```

Comment:
Will have to move these styles to the new styling convention (styled components). If it's too much for this pr, we can def refactor it in a future pr.

Reply:
yes, different pr pls